### ISSUE TITLE
Template clone can fail in windows machines

### ISSUE THREAD

**Issue by stevejpurves** [executablebooks/mystmd#209]
### What version of `mystjs` are you using?

0.1.11

### How did you install myst?

npm

### What operating system are you using?

Windows

### Which area is this feature request for?

Command Line Interface

### Describe the Bug

During `myst start` a default `book-theme template cloned onto the user's local machine. This can fail on some windows machines but succeed on others. The machine that fails has an up-to-date `git` installation accessible on the path. 
An excerpt of the error message is:

```
📚 Built 12 pages for Eindrapport in 36 s.
🔍 Querying template metadata from https://api.curvenote.com/templates/site/myst/book-theme
🌎 Cloning template from https://github.com/curvenote/book-theme.git
fatal: Too many arguments.

usage: git clone [<options>] [--] <repo> [<dir>]

...

Command failed: git clone --recursive --depth 1 --branch main https://github.com/executablebooks/myst-book-theme.git _build\templates\site\myst\book-theme
fatal: Too many arguments
```

If the user then runs the git command independently in the same command prompt window, the theme is correctly cloned, and a subsequent `myst start` succeeds.

### Expected Behavior

The `git clone` of the template should succeed when issues by the `myst` cli.

### To Reproduce

Start from scratch on a windows machine with a fresh `myst` project.

Note: I am reporting this as another user had this error persistently and I tried to help them with it. HOwever, I have not been able to reproduce this on a Windows 10 machine.

**Comment by rowanc1** [comment]
cc @choldgraf looks like you ran into this, if you have time to let one of us debug this with you that would be awesome.

**Comment by choldgraf** [comment]
Note that this wasn't a windows machine per se, it was WSL within Windows. I'm pretty busy this whole week unfortunately but maybe next week?

**Comment by fwkoch** [comment]
I think for this, we just take out the `git clone` and instead download the zip from github directly.

Previously we needed a recursive clone because the themes required `myst-theme` as a submodule. Now, though, the themes are built into nice standalone bundles, so the zip will work.

**Comment by rowanc1** [comment]
I believe this should be fixed in the next version @choldgraf. We have switched to zip download and just avoided the code path. I will let you know when it is live!

**Comment by rowanc1** [comment]
This has been released in v1.1.4!
